native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -149,6 +150,25 @@ static char * normalize_hostname(apr_pool_t *p, char *hostname)
     }
     return ret;
 }
+/**
+ * Normalize the worker name
+ */
+static char* normalize_workername(apr_pool_t *pool, char *url) {
+    char *ptr;
+    apr_uri_t uri;
+    if (apr_uri_parse(pool, url, &uri) != APR_SUCCESS) {
+        return NULL;
+    }
+    if (!uri.scheme) {
+        return NULL;
+    }
+    if (uri.port && uri.port == ap_proxy_port_of_scheme(uri.scheme)) {
+        uri.port = 0;
+    }
+    ptr = apr_uri_unparse(pool, &uri, APR_URI_UNP_REVEALPASSWORD);
+    return ptr;
```

Comment:
I think the `ptr` can be dropped as it is not used – `return apr_uri_unparse(...);` should do the job.